### PR TITLE
feat: future setup for support other l2

### PR DESCRIPTION
**Pull request by solidovic**
### Description

- refactor(wrap/unwrap): preparing FAQ for other networks
- refactor(etherscan): helpers

### Checklist:

- [x] Checked the changes locally.
- [ ] Created / updated analytics events.
- [ ] Created / updated the technical documentation (README.md / [docs](https://docs.lido.fi/) / etc.).
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: utils/etherscan.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export const getEtherscanLink = (
+  chainId: number,
+  hash: string,
+  entity: ETHERSCAN_ENTITIES,
+): string => {
+  const wagmiChain = wagmiChainMap[chainId] ?? mainnet;
+  return wagmiChain?.blockExplorers?.default?.url
+    ? `${wagmiChain.blockExplorers.default.url}/${entity}/${hash}`
+    : // Most likely this fallback will never work
+      `https://etherscan1.io/${entity}/${hash}`;
```
Comment (Jeday) [inline utils/etherscan.ts:19]: WTF is this domain???

Comment (Jeday) [reply utils/etherscan.ts:19]: Also let's just do invairaint here. We won't have non-wagmi chain.

File: features/wsteth/shared/wrap-faq/wrap-faq.tsx [hunk 2]
```diff
@@ -1,56 +1,28 @@
...
   return (
     <Section title="FAQ" onClick={onClickHandler}>
-      <WhatIsWsteth />
-      <HowCanIGetWsteth />
-      <HowCanIUseWsteth />
-      <DoIGetMyStakingRewards />
-      <DoINeedToClaimMyStakingRewards />
-      <HowCouldIUnwrapWstethToSteth />
-      <DoINeedToUnwrapMyWsteth />
+      {React.createElement(faqComponent, { key: chainType })}
```
Comment (Jeday) [inline features/wsteth/shared/wrap-faq/wrap-faq.tsx:25]: Just use UpperCase for it be jsx.

File: shared/components/tx-link-etherscan/tx-link-etherscan.tsx [hunk 3]
```diff
@@ -12,14 +12,14 @@ type TxLinkEtherscanProps = {
 
 export const TxLinkEtherscan = (props: TxLinkEtherscanProps) => {
   const { txHash, text = 'View on Etherscan', onClick } = props;
-  const { chainId } = useDappStatus();
+  const { walletChainId } = useDappStatus();
```
Comment (Jeday) [inline shared/components/tx-link-etherscan/tx-link-etherscan.tsx:15]: Why use walletChainId? If we don't support chain we should not try to link correct explorer, just use chainId(it defaults to correct chain)

**Review by Jeday — CHANGES_REQUESTED** [review]

File: shared/components/tx-link-etherscan/tx-link-etherscan.tsx (hunk 3 above)
Comment (solidovic) [reply shared/components/tx-link-etherscan/tx-link-etherscan.tsx:15]: image
This component is used in TransactionModal, which is wrapped by SupportL1Chains, but not wrapped by SupportL2Chains.
This is currently the fastest solution

File: features/wsteth/shared/wrap-faq/wrap-faq.tsx (hunk 2 above)
Comment (solidovic) [reply features/wsteth/shared/wrap-faq/wrap-faq.tsx:25]: A rudiment remained after coding

**Review by Jeday — DISMISSED** [review]

File: utils/etherscan.ts (hunk 1 above)
Comment (Jeday) [reply utils/etherscan.ts]: I freaked out because it's apparently a fishing domain.

**Review by Jeday — APPROVED** [review]

**Pull request merged**